content/en/apps/guides/integrations/best-practices/rapidpro.md

```diff
@@ -0,0 +1,108 @@
+---
+title: RapidPro best practices and lessons from integration with the CHT
+linkTitle: "RapidPro"
+weight: 20
+description: >
+    Best practices for RapidPro deployments
+keywords: rapidpro, best practices, tips, lessons, integration
+relatedContent: >
+    apps/features/integrations/rapidpro
+    apps/guides/integrations/rapidpro
+---
+
+
+### Introduction
+[RapidPro](https://app.rapidpro.io/) is the Open Source version of Textit, developed by UNICEF and Nyaruka, to  visually build workflow logic targeting mobile-based services. Here, we detail the best practices when working on a RapidPro deployment. Top of the list is to review RapidPro’s documentation to familiarize with various components including the [API](https://rapidpro.io/api/v2/). Next, we focus on various components or stages in the deployment process.
+
+### [Workspaces](https://rapidpro.github.io/rapidpro/docs/orgs/)
+Think of workspaces as the equivalent of an instance in the CHT. Medic has an internally hosted RapidPro. Ensure that you request for both `staging` and `production` workspaces early enough to allow SRE set up the workspaces on time.
+
+### SMS Messaging Channels
+- RapidPro supports both Android channels and SMS aggregators. Note that Android channels have a messaging limit of 330 outgoing messages/hour. If using Android channels, ensure that you install the maximum number of SMS packs and follow the following best practices for best results:
+   1. Avoid the Android Channel going to sleep 
+    The following apply (based on Samsung A7 2018) with more details available on [dontkillmyapp](https://dontkillmyapp.com/samsung):
```

Comment:
should we just link to https://dontkillmyapp.com's main page instead of their samsung page?  it provides recommendations for all sorts of phone varieties and i'm not sure it is safe to assume people would use an android

Reply:
Make sense.